README.md

```diff
@@ -28,12 +28,14 @@ $ bash Miniconda3-latest-Linux-x86_64.sh
 ```shell
 $ git clone https://github.com/hannbus/ChemIcal_DatasEt_compaRator.git
 $ cd ChemIcal_DatasEt_compaRator
-$ conda create --name ChemIcal_DatasEt_compaRator python=3.10
-$ conda activate ChemIcal_DatasEt_compaRator
+$ conda create --name cider_chem python=3.10
+$ conda activate cider_chem
 $ conda install pip
 $ python -m pip install -U pip #Upgrade pip
 $ pip install .
 ```
+- Note: Make sure all installations are working correctly by running the tests.
```

Comment:
I would add: ... by running the "pytest" command in the repository root folder.

Reply:
Modified.